src/appsignal/cli/diagnose.py

```diff
@@ -0,0 +1,390 @@
+        if self.agent_report.configuration_error():
+            print(f"     Error: {self.agent_report.configuration_error()}")
+        print("  Agent tests")
+        print(f"    Started: {self.agent_report.started()}")
+        print(f"    Process user id: {self.agent_report.user_id()}")
+        print(f"    Process user group id: {self.agent_report.group_id()}")
+        print(f"    Configuration: {self.agent_report.configuration_valid()}")
+        print(f"    Logger: {self.agent_report.logger_started()}")
+        print(
+            f"    Working directory user id: {self.agent_report.working_directory_user_id()}"
+        )
+        print(
+            f"    Working directory user group id: {self.agent_report.working_directory_group_id()}"
+        )
+        print(
+            f"    Working directory permissions: {self.agent_report.working_directory_permissions()}"
+        )
+        print(f"    Lock path: {self.agent_report.lock_path()}")
+
+    def _configuration_information(self) -> None:
+        print("Configuration")
+
+        for key in self.config.options:
+            print(f"  {key}: {self.config.options[key]!r}")  # type: ignore
+
+        print()
+        print("Read more about how the diagnose config output is rendered")
+        print("https://docs.appsignal.com/python/command-line/diagnose.html")
+
+    def _validation_information(self) -> None:
+        validation_report: Any = self.report["validation"]
+        print("Validation")
+        print(f'  Validating Push API key: {validation_report["push_api_key"]}')
+
+    def _paths_information(self) -> None:
+        log_file_path = self.config.log_file_path() or ""
+        log_path = self.config.option("log_path") or os.path.dirname(log_file_path)
+
+        try:
+            with open(log_file_path) as log_file:
+                lines = log_file.readlines()
+                log_file_contents = lines[-10:]
+        except FileNotFoundError:
+            log_file_contents = [""]
+
+        print("Paths")
+        print("  Current working directory")
+        print(f"    Path: '{os.getcwd()}'")
+        print()
+        print("  Log directory")
+        print(f"    Path: '{log_path}'")
+        print()
+        print("  AppSignal log")
+        print(f"    Path: '{log_file_path}'")
+
+        print("    Contents (last 10 lines):")
+        for line in log_file_contents:
+            print(line.strip())
+
+    def _report_information(self) -> None:
+        print("Diagnostics report")
+
+    def _report_prompt(self) -> bool | None:
+        print("  Do you want to send this diagnostics report to AppSignal?")
+        print("  If you share this report you will be given a link to")
+        print("  AppSignal.com to validate the report.")
+        print("  You can also contact us at support@appsignal.com")
+        print("  with your support token.")
+        print()
+
+        send_report = input("  Send diagnostics report to AppSignal? (Y/n):   ")
```

Comment:
```suggestion
        send_report = input("  Send diagnostics report to AppSignal? (Y/n): ")
```

Extra spaces unnecessary?

Reply:
That's what the diagnose-tests expect:

```
"  Send diagnostics report to AppSignal? (Y/n):   Transmitting diagnostics report",
```